src/app/project/board/project-tab-view/project-tab-view.component.html

```diff
@@ -0,0 +1,94 @@
+                        </mat-chip-list>
+                    </div>
+
+                    <!-- #################### template for dataManagementPlan ################# -->
+                    <div *ngSwitchCase="'dataManagementPlan'">
+                        <p style="margin-top: 4px;">
+                            URL: <a href="{{ metadata['dataManagementPlan']['url']['value'] }}" target="_blank"> {{ metadata['dataManagementPlan']['url']['value'] }} </a>
+                        </p>
+                        <p>isAvailable: {{ metadata['dataManagementPlan']['isAvailable'] }}</p>
+                    </div>
+
+                    <!-- #################### template for dataManagementPlan ################# -->
+                    <div *ngSwitchCase="'grant'">
+
+                        <div class="person">
+                            <p> {{ metadata['grant']['name'] }} </p>
+                            <div *ngIf="metadata['grant']['number']" class="mat-icon-text">
+                                <mat-icon aria-hidden="false" aria-label="Call" class="small">call</mat-icon>
+                                <a href="tel:{{ metadata['grant']['number'] }}"> {{ metadata['grant']['number'] }} </a>
+                            </div>
+
+                            <div *ngIf="metadata['grant']['url']['value']">
+                                <h4>URL:</h4>
+                                <a href="{{ metadata['grant']['url']['value'] }}" target="_blank"> {{ metadata['grant']['url']['value'] }} </a>
+                            </div>
+
+                            <div class="sub-details">
+                                <h4>Funder(s):</h4>
+                                <mat-card *ngFor="let entry of metadata['grant']['funder'] | keyvalue;" class="funder-entry">
+                                    <div *ngIf="metadata['grant']['funder'][entry.key]['name']">
+                                        <app-organisation-template  [organisation]="metadata['grant']['funder'][entry.key]"></app-organisation-template>
+                                    </div>
+                                    <div *ngIf="metadata['grant']['funder'][entry.key]['givenName']">
+                                        <app-person-template [person]="metadata['grant']['funder'][entry.key]"></app-person-template>
+                                    </div>
+                                </mat-card>
+                            </div>
+                        </div>
+
+                    </div>
+
+                    <!-- #################### template for dataManagementPlan ################# -->
+                    <div *ngSwitchCase="'spatialCoverage'">
+                        <div *ngFor="let coverage of metadata['spatialCoverage'] | keyvalue;">
+                            <a href="{{ coverage.value['place']['url'] }}" target="_blank"> {{ coverage.value['place']['name'] }} </a>
```

Comment:
Instead of displaying object, for now I am displaying it as: 
`coverage.value['place']['name'] ( coverage.value['place']['url'] )`

For example:
Geonames ( https://www.geonames.org/2017370/russian-federation.html )